content/engine/release-notes/25.0.md

```diff
@@ -0,0 +1,103 @@
+- containerd image store: Add support for pulling legacy schema1 images. [moby/moby#46513](https://github.com/moby/moby/pull/46513)
+- containerd image store: Add support for pushing all tags. [moby/moby#46485](https://github.com/moby/moby/pull/46485)
+- containerd image store: Add support for registry token. [moby/moby#46475](https://github.com/moby/moby/pull/46475)
+- containerd image store: Add support for showing the number of containers that use an image. [moby/moby#46511](https://github.com/moby/moby/pull/46511)
+- containerd image store: Fix `ONBUILD`, `MAINTAINER`, and healthchecks in Dockerfile instruction for support. [moby/moby#46313](https://github.com/moby/moby/pull/46313)
+- containerd image store: Fix `Pulling from` progress message. [moby/moby#46494](https://github.com/moby/moby/pull/46494)
+- containerd image store: Fix `TestInspectByPrefix` [moby/moby#46435](https://github.com/moby/moby/pull/46435)
+- containerd image store: Fix `docker images` showing intermediate layers by default. [moby/moby#46423](https://github.com/moby/moby/pull/46423)
+- containerd image store: Fix checking if the specified platform exists when getting an image. [moby/moby#46495](https://github.com/moby/moby/pull/46495)
+- containerd image store: Fix errors when multiple `ADD` or `ADD` instructions were used with the classic builder. [moby/moby#46383](https://github.com/moby/moby/pull/46383)
+- containerd image store: Fix stack overflow errors when importing an image. [moby/moby#46418](https://github.com/moby/moby/pull/46418)
+- containerd image store: Improve `docker pull` progress output. [moby/moby#46412](https://github.com/moby/moby/pull/46412)
+- containerd image store: Print the tag, digest, and size after pushing an image. [moby/moby#46384](https://github.com/moby/moby/pull/46384)
+- containerd image store: Remove panic from `UpdateConfig` [moby/moby#46433](https://github.com/moby/moby/pull/46433)
+- containerd image store: Return an error when an image tag resembles a digest. [moby/moby#46492](https://github.com/moby/moby/pull/46492)
+
+### Packaging updates
+
+- Upgrade API to v1.44. [moby/moby#45468](https://github.com/moby/moby/pull/45468)
+- Upgrade containerd to v1.7.6. [moby/moby#46486](https://github.com/moby/moby/pull/46486)
+- Upgrade go to v1.20.8. [docker/cli#4557](https://github.com/docker/cli/pull/4557), [moby/moby#46453](https://github.com/moby/moby/pull/46453)
+- Upgrade runc to v1.1.9. [moby/moby#46192](https://github.com/moby/moby/pull/46192)
+- Upgrade swarmkit to v2.0.0. [docker/cli#4408](https://github.com/docker/cli/pull/4408), [moby/moby#46184](https://github.com/moby/moby/pull/46184)
+
+### Removed
+
+- API: Remove VirtualSize field for the `GET /images/json` and `GET /images/{id}/json` endpoints. [moby/moby#45469](https://github.com/moby/moby/pull/45469)
+- Remove `ro-non-recursive`, `ro-force-recursive`, and `rro` from the legacy mount API. [moby/moby#46037](https://github.com/moby/moby/pull/46037)
```

Comment:
added and removed rro in the same release 🤔

Reply:
We tweaked the original PR to not expose things via the old API; we should squash these into a single like ("add RRO").